pkg/reading/names.go

```diff
@@ -0,0 +1,29 @@
+package reading
+
+import (
+	"encoding/json"
+	"errors"
+)
+
+type Names struct {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported type Names should have comment or be unexported](https://app.codacy.com/app/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4570750)